apps/site/pages/courses/[courseSlug]/index.tsx

```diff
@@ -293,6 +306,16 @@ function CourseHeader({
 							<PlusCircleIcon className="h-5" />
 						</button>
 					)}
+
```

Comment:
quasi duplicate code.

evtl authentication check in onClick verlagern? So könnte dort eine überprüfung stattfinden. 
Beispiel: 
```ts
	const enrollOrRedirect = () => {
		if (isUserAuthenticatedInSession) {
			enroll({ courseId: course.courseId });
		} else {
			redirectToLogin();
		}
	};
```

`onClick={enrollOrRedirect}`